src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -28,7 +28,7 @@
     {{ form_row(addProductRowForm.product_id) }}
     {{ form_row(addProductRowForm.tax_rate) }}
     {{ form_label(addProductRowForm.search) }}
-    {{ form_widget(addProductRowForm.search) }}
+    {{ form_widget(addProductRowForm.search, { 'attr': {'data-currency': orderForViewing.currencyId }}) }}
```

Comment:
Could you add this in the `AddProductRowType` instead?
In the next version (and a little in this one) we are trying to fully fill the form types in order to simplify the twig template with a simple `from_widget(addProductRowForm)` but this requires to correctly create the form beforehand

In `OrderController::viewAction` when the form is created you can inject the currency id as an option, which will then allow you to add this field on the `search` form field Of course what you did works and we may improve it later, but if it's not too long to do it now it will save us a bit of time later

Reply:
Done